src/js/components/SearchInput.js

```diff
@@ -115,19 +115,21 @@ export default class SearchInput extends Component {
   _onAddDrop (event) {
     event.preventDefault();
     // Get values of suggestions, so we can highlight selected suggestion
-    let suggestionValues = this.props.suggestions.map((suggestion) => {
+    if (this.props.suggestions) {
+      let suggestionValues = this.props.suggestions.map((suggestion) => {
       if (typeof suggestion === 'object') {
-        return suggestion.value;
```

Comment:
Change looks good, thanks for your contribution.

One small request, in grommet we are using two characters spacing. would you mind to fix the spacing in this change?